src/components/Modal.js

```diff
@@ -44,7 +41,11 @@ export const Modal = ({
 
 const styles = StyleSheet.create({
   overlay: {
+    borderRadius: normalize(8),
```

Comment:
why is this changed to be normalized? do we need this? can you find some changes when trying different values or even no border radius? i cannot see differences.

Reply:
`borderRadius` style added to match the previous design

|without `borderRadius`|with `borderRadius`|
|--|--|
![IMG_2632](https://user-images.githubusercontent.com/11755668/222380193-067c4d3a-0a57-4eba-b865-063514733cfb.jpg) | ![IMG_2631](https://user-images.githubusercontent.com/11755668/222380216-5b3cecb9-0b9f-4229-81e2-4838dc2843ca.jpg)